manifests/init.pp

```diff
@@ -117,5 +70,4 @@
   class { '::ntp::config': } ~>
   class { '::ntp::service': } ->
   anchor { 'ntp::end': }
```

Comment:
IMHO the refresh of services when configs changes should be a behaviour, enabled by default, that users must be able to change, via a class parameter. There can be situations (probably it's not the case for ntpd, but can be valid for other services) where an immediate restart of the service is not desired.